### DISCUSSION TITLE
Smoke detector GS559AHHK

### DISCUSSION THREAD

**Discussion by webwude** [krahabb/meross_lan#282] [General]
Hi.

I have an addtional new device - a smoke detector. It will be recognized somehow as well - but I guess not completely:

![grafik](https://user-images.githubusercontent.com/7056632/233993263-25a565eb-6380-4f3b-81a7-c55463b149c7.png)

I can set a test alarm - and create a diagnostic trace during that, if this helps. The switch and the other attributes don't seem to report anything so far.

best
w.

**Comment by krahabb** [comment]
Yep, there's an opened 'issue' at #183 where it was discussed a bit.
At the moment I'm still reluctant to add more 'complexity' to the `meross_lan` implementation until we get a more definitive device behavior description (even though [this](https://github.com/krahabb/meross_lan/issues/183#issuecomment-1366733115) looks as a good starting point)

**Comment by webwude** [comment]
Thx for pointing to the issue. Looks like a good start for me too.
